spec/system/hiring_staff_can_add_stats_to_a_vacancy_spec.rb

```diff
@@ -24,12 +24,6 @@
       third_vacancy.organisation_vacancies.create(organisation: school)
     end
 
-    scenario "hiring staff can see notification badge" do
-      visit organisation_path
-
-      expect(page).to have_selector(NOTIFICATION_BADGE_SELECTOR, text: 3)
```

Comment:
Are we removing the 'awaiting feedback' notifications?
If so, please can you remove the other two instances of `NOTIFICATION_BADGE_SELECTOR`

Reply:
will address this, but its imminent we will be removing the awaiting feedback stuff altogether

Reply:
when chris gives the thumbs up, ticket in backlog already